src/main/java/network/kngsgaming/minecraft/EnchantEvents.java

```diff
@@ -221,16 +220,17 @@ public void onInventoryClick(InventoryClickEvent event){
     }
 
     //Check to see if this enchantment and level are within the config defined limits
+    //Unused?
     private boolean isValidEnchantLevel(Enchantment enchantment, int level) {
         //assume no limit!
         int limit = Integer.MAX_VALUE;
         String enchanmentName = enchantment.getKey().getKey();
         if (limits.containsKey(enchanmentName)) {
-            limit  = 0;
```

Comment:
Moving this into the catch is great! it saves an extra assignment if the try works.